src/common/options/cephfs-mirror.yaml.in

```diff
@@ -80,3 +80,15 @@ options:
   services:
   - cephfs-mirror
   min: 0
+- name: cephfs_mirror_fs_mount_timeout
+  type: secs
+  level: advanced
+  desc: timeout for mounting primary/seconday ceph file system
+  long_desc: Timeout interval in seconds use by the mirror daemon when mounting primary
+    or secondary (remote) ceph file system. Setting this to a higher value could result
+    in the mirror daemon getting stalled when mounting a file system if the cluster is
+    not reachable.
+  default: 10
+  services:
+  - cephfs-mirror
+  min: 0
```

Comment:
what does `0` mean? Infinite?

Reply:
Basically, it mean abort on the next tick (Client::tick) if mount is still ongoing as per:

```
    if (req->op_stamp + cct->_conf->client_mount_timeout < now) {
      req->abort(-CEPHFS_ETIMEDOUT);
```